language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>I'd also split this into 3 parts: before/prompt template/after.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>cheers - updated by Alicia</p>